### DISCUSSION TITLE
How to trigger run of a MAP on AWS Sagemaker?

### DISCUSSION THREAD

**Discussion by KnightCoder** [Project-MONAI/monai-deploy-app-sdk#459] [Q&A]
Using MONAI Deploy version 0.5.1 I've packaged my app into a MAP and it contains all the model files within itself. 

Can I shoot a REST API call (from internet) that will trigger the execution of a MAP on AWS Sagemaker? If yes, please share the instructions /article / doc explaining steps to achieve this. At the moment I couldn't find anything that helped with this.

**Comment by MMelQin** [comment]
Thank you @KnightCoder for the question.

There exists a folder in this repo [here](https://github.com/Project-MONAI/monai-deploy-app-sdk/tree/main/platforms/aws_healthimaging), showing the couple integrations with AWS HealthImaging API and SageMaker. Specifically, the [link ](https://github.com/aws-samples/monai-on-aws-workshop) leads to the Jypyter notebook on SageMaker integration.

More will come to make the integration more efficient.
